### ISSUE TITLE
librarycarpentry/lc-git

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#63]
tracking issue for <https://github.com/librarycarpentry/lc-git>

- [x] problems caused by [stray kramdown source tag](https://github.com/LibraryCarpentry/lc-git/blob/1065f14150e0710c8d741f5cde46159bdbe09304/_episodes/02-getting-started.md?plain=1#L95) (maybe fixed in https://github.com/LibraryCarpentry/lc-git/pull/136)

<details><summary>Details</summary>
<p>

```r
── Reading in lesson with pegboard ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Error in `purrr::map()`:
ℹ In index: 2.
ℹ With name: /home/zhian/Documents/Carpentries/Git/carpentries/lesson-transition/librarycarpentry/lc-git/_episodes/02-getting-started.md.
Caused by error in `purrr::map()`:
ℹ In index: 1.
Caused by error in `xml_nodeset()`:
! Expecting an external pointer: [type=NULL]
Backtrace:
     ▆
  1. ├─pegboard::Lesson$new(old, fix_liquid = arguments$fix_liquid)
  2. │ └─pegboard (local) initialize(...)
  3. │   └─pegboard:::read_jekyll_episodes(path, rmd, ...)
  4. │     └─pegboard:::read_markdown_files(md_src, sandpaper = FALSE, ...)
  5. │       └─purrr::map(the_episodes, Episode$new, ...)
  6. │         └─purrr:::map_("list", .x, .f, ..., .progress = .progress)
  7. │           ├─purrr:::with_indexed_errors(...)
  8. │           │ └─base::withCallingHandlers(...)
  9. │           ├─purrr:::call_with_cleanup(...)
 10. │           └─R6 (local) .f(.x[[i]], ...)
 11. │             └─pegboard (local) initialize(...)
 12. │               └─purrr::map(blocks, set_ktag_block)
 13. │                 └─purrr:::map_("list", .x, .f, ..., .progress = .progress)
 14. │                   ├─purrr:::with_indexed_errors(...)
 15. │                   │ └─base::withCallingHandlers(...)
 16. │                   ├─purrr:::call_with_cleanup(...)
 17. │                   └─pegboard (local) .f(.x[[i]], ...)
 18. │                     ├─parents[tag]
 19. │                     └─xml2:::`[.xml_nodeset`(parents, tag)
 20. │                       └─xml2:::xml_nodeset(NextMethod())
 21. └─base::.handleSimpleError(...)
 22.   └─purrr (local) h(simpleError(msg, call))
 23.     └─cli::cli_abort(...)
 24.       └─rlang::abort(...)
Execution halted
```

</p>
</details>

**Comment by zkamvar** [comment]
The lesson is ready in https://fishtree-attempt.github.io/lc-git/

**Comment by zkamvar** [comment]
fixed in f3bd295
